.github/workflows/nightly.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish nightly build artifacts
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghidra: ["9.2", "9.2.1", "9.2.2", "9.2.3", "9.2.4"]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11"
+          java-package: jdk
+          architecture: x64
+      - uses: er28-0652/setup-ghidra@master
+        with:
+          version: ${{ matrix.ghidra }}
+      - uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-version: 6.9
```

Comment:
Not sure if this is needed or not: https://github.com/gradle/gradle-build-action#setup-and-use-a-declared-gradle-version
> The Gradle wrapper's version (default, useful for matrix builds)
```suggestion
```